addons/stock/static/src/views/search/stock_orderpoint_search_model.js

```diff
@@ -9,21 +9,24 @@ export class StockOrderpointSearchModel extends SearchModel {
     setup(services) {
         super.setup(services);
         this.ui = useService("ui");
-        this.applyGlobalVisibilityDays = debounce(
-            this.applyGlobalVisibilityDays.bind(this),
+        this.applyGlobalHorizonDays = debounce(
+            this.applyGlobalHorizonDays.bind(this),
             StockOrderpointSearchModel.DEBOUNCE_DELAY
         );
     }
 
-    async applyGlobalVisibilityDays(globalVisibilityDays) {
+    async applyGlobalHorizonDays(globalHorizonDays) {
         this.ui.block();
         this.globalContext = {
             ...this.globalContext,
-            global_visibility_days: globalVisibilityDays,
+            global_horizon_days: globalHorizonDays,
         };
         this._context = false; // Force rebuild of this.context to take into account the updated this.globalContext
         await this.orm.call("stock.warehouse.orderpoint", "action_open_orderpoints", [], {
-            context: this.context,
+            context: {
+                ...this.context,
+                force_orderpoint_recompute: true,
```

Comment:
My spider sense is tingling that this forcing of the recomputes for all orderpoints is bad, but need to dig more into why. Calling `action_open_orderpoints` previously would recompute a lot of stuff, does it not do it sufficiently for the this new `global_horizon_days` like it used to for the `global_visibility_days`?